build.sh

```diff
@@ -111,4 +111,4 @@ if [ $RUN_IN_CI == 0 ]; then
   # TODO Temporarily disabling ios build
   #rm -rf ThaliTest.app;ERROR_ABORT
   #cp -R ../ThaliTest/platforms/ios/build/device/ThaliTest.app ThaliTest.app;ERROR_ABORT
-fi
+fi
```

Comment:
@vasilevskayaem you shouldn't remove this __line__, but remove your __change__, since it's unnecessary.